### PR TITLE
fix memory leak

### PR DESCRIPTION
**Pull request by rabauke**

### CODE REVIEW

File: src/lua/events.c [hunk 1]
```diff
@@ -628,6 +628,10 @@ static int destroy_shortcut_event(lua_State *L)
 
   // remove the accelerator from the lua shortcuts
   dt_accel_deregister_lua(tmp);
+
+  // free temporary buffer
+  free(tmp);
+
```
Comment (rabauke) [inline src/lua/events.c:634]: Buffer has been allocated by `strdup` a few lines above. Cannot find where it is freed.

**Comment by TurboGit** [comment]
@wpferguson : Can you review this one as part of Lua code ? TIA.

File: src/lua/events.c (hunk 1 above)
Comment (wpferguson) [reply src/lua/events.c:634]: This works fine.  You can add a free in the routine above, after line 606, to free that tmp also.

**Comment by wpferguson** [comment]
@TurboGit I tested and it works fine.  I also removed the strdup tmp in the routine above, register_event_shortcut, tested it and that worked fine also so added a comment for @rabauke to fix that one too while he was in there.

File: src/lua/events.c (hunk 1 above)
Comment (rabauke) [reply src/lua/events.c:638]: @wpferguson Thanks for reviewing and for pointing out the other leak. I revised my pr to fix both.

Comment (wpferguson) [reply src/lua/events.c:638]: Thanks for fixing :)

**Comment by wpferguson** [comment]
@TurboGit this is good to go

**Review by TurboGit — APPROVED** [review]
Thanks for both of you!

**Pull request merged**